common/reviews/api/heft.api.md

```diff
@@ -54,6 +54,8 @@ export class CleanStageHooks extends StageHooksBase<ICleanStageProperties> {
 export class CompileSubstageHooks extends BuildSubstageHooksBase {
     // (undocumented)
     readonly afterCompile: AsyncParallelHook;
+    // (undocumented)
+    readonly afterEachIteration: SyncHook;
```

Comment:
@iclanton What's the right way to model this notification?

It's wired up to this:
```ts
  /**
   * Fired whenever the compiler emits an output.  In watch mode, this event occurs after each recompile.
   * If there are multiple tsconfigs being processed in parallel, the event fires for each one.
   */
  emitCallback: () => void;
```

Reply:
Yeah I think that looks fine.